lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
@simon3z No, port name (or number - both are acceptable) are definitely not required for single port services.